features/user_can_delete_message.feature

```diff
@@ -0,0 +1,25 @@
+Feature: User can empty inbox
+  As a user
+  In order to clean up my inbox
+  I want to be able to delete messages
+
+  Background:
+    Given the following user exist:
+
+      | name   | email          | password  | id |
+      | dmybob | dmybob@bob.bob | bobbobbob | 1  |
+      | dmypip | dmypip@bob.bob | pippippip | 2  |
+
+    Given the inbox has content
```

Comment:
please change to `And the inbox has content`

Reply:
ok